tracing-subscriber/CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+# 0.3.18 (November 6, 2023)
+
+This release of `tracing-subscriber` adds support for `NO_COLOR` in
```

Comment:
let's explicitly state that `NO_COLOR` is an environment variable? for a reader who's unfamiliar with the env var, it's unclear what "NO_COLOR support" means. maybe link to https://no-color.org/ as well?

Reply:
done!

Reply:
rewrote the introduction and linked to it.